src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBSCToSirius.vue

```diff
@@ -530,20 +570,13 @@ export default {
     });
 
     const amountReceived = computed(() => {
-      if((amount.value - 50) >0 ){
-        return (amount.value - 50);
+      if(Math.round((amount.value - feeAmount.value)*Math.pow(10,tokenDivisibility.value))/Math.pow(10,tokenDivisibility.value) >0 ){
+        return Math.round((amount.value - feeAmount.value)*Math.pow(10,tokenDivisibility.value))/Math.pow(10,tokenDivisibility.value)
```

Comment:
I don't think you need to round it, just convert both to base unit, deduct it then divide the result